managed/services/management/dbaas/kubernetes_server.go

```diff
@@ -490,25 +517,68 @@ func (k kubernetesServer) installDefaultOperators(operatorsToInstall map[string]
 		if !ok || channel == "" {
 			channel = "stable-v0"
 		}
-		params := kubernetes.InstallOperatorRequest{
-			Namespace:              namespace,
-			Name:                   operatorName,
-			OperatorGroup:          operatorGroup,
-			CatalogSource:          "percona-dbaas-catalog",
-			CatalogSourceNamespace: catalogSourceNamespace,
-			Channel:                channel,
-			InstallPlanApproval:    v1alpha1.ApprovalManual,
-		}
+		_, err := kubeClient.GetSubscription(ctx, namespace, operatorName)
```

Comment:
I like the idea to get a subscription first but I think that we need to get CSV and check that it has the 'Installed' or 'Succeeded' phase in the status field. Getting a CSV object is a better option to understand that operator is installed. 

As an example of the corner case that might be in the cluster. Let's imagine that we have two channels fast and stable (or, let's say that we have one channel per PMM's release). And dbaas-catalog wasn't installed, so there will be a subscription present in the cluster and it uses as a source percona-dbaas-catalog, yet CSV won't be created because dbaas-catalog is not present in the cluster. 

Getting the state of csv (and this is what we need to have in the end) will tell us more about the state of the operator's installation. What do you think about moving from GetSubscriotions to GetCSV?

Reply:
by ` yet CSV won't be created because dbaas-catalog is not present in the cluster.` you wanted to say subscription, right?